### PR TITLE
Spike that tests google search page.

### PR DESCRIPTION
**Pull request by jmosley5**

### CODE REVIEW

File: searchTest.js [hunk 1]
```diff
@@ -1,41 +1,35 @@
...
-test('Search for AOEU', async t => {
     await t
-        //Type "The Art of Education University in the text field"
-        .typeText('.gLFyf', 'The Art of Education University')
-        //Clicks the google search button
-        .click('.gNO89b')
-        //Clicks "The Art of Education University" link
+        .typeText(googleSearchBox, 'The Art of Education University')
+        .click(googleSearchButton)
         .click('.yuRUbf')
```
Comment (Thomas-Jean) [inline searchTest.js]: could also give `.yuRUbf` a more meaningful name like` googleSearchResult`?

Comment (jmosley5) [reply searchTest.js]: Yep! Thank you.

**Review by Thomas-Jean — APPROVED** [review]

**Review by ryexley — APPROVED** [review]

**Comment by theantichris** [comment]
@jmosley5 You can merge this PR.

**Pull request merged**